cron/k8s/controller.release.yaml

```diff
@@ -52,7 +52,11 @@ spec:
           containers:
             - name: controller
               image: gcr.io/openssf/scorecard-batch-controller:latest
-              args: ["--config=/etc/scorecard/config.yaml", "cron/internal/data/projects.release.csv"]
+              args: [
+                "--config=/etc/scorecard/config.yaml",
+                "cron/internal/data/projects.release.csv",
+                "cron/internal/data/gitlab-projects-selected.csv"
```

Comment:
I still get panics running this against repos in this list. Lets fix these up before merging in.